<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTreeWidget" name="treeSlaves">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>1666254</height>
           </size>
          </property>
          <property name="title">
           <string>Information</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_name">
             <property name="text">
              <string>Name :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_name"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_index">
             <property name="text">
              <string>Index :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_index"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_current_value">
             <property name="text">
              <string>Current Value :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_current_value"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_target_value">
             <property name="text">
              <string>Target Value :</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_target_value"/>
           </item>
           <item row="5" column="0">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>OK</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTextBrowser" name="Info_Browser">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>120</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="StayOpenMenu" name="menuAdapter_A">
    <property name="title">
     <string>Adapter(&amp;A)</string>
    </property>
    <addaction name="actionScan_Adapter"/>
   </widget>
   <widget class="StayOpenMenu" name="menuSlave_S">
    <property name="title">
     <string>Slave(&amp;S)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuProgram_P">
    <property name="title">
     <string>Program(&amp;P)</string>
    </property>
   </widget>
   <addaction name="menuAdapter_A"/>
   <addaction name="menuSlave_S"/>
   <addaction name="menuProgram_P"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="state_indicator_bar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_OP"/>
   <addaction name="separator"/>
   <addaction name="action_Safe_OP"/>
   <addaction name="separator"/>
   <addaction name="action_Pre_OP"/>
   <addaction name="separator"/>
   <addaction name="action_Init"/>
   <addaction name="separator"/>
   <addaction name="action_Boot_Strap"/>
  </widget>
  <action name="actionScan_Adapter">
   <property name="text">
    <string>Scan Adapter</string>
   </property>
   <property name="statusTip">
    <string>Scan for available Network Adapter</string>
   </property>
  </action>
  <action name="action_Init">
   <property name="icon">
    <iconset resource="Icon_resource.qrc">
     <normaloff>:/my_images/icon/grey light.png</normaloff>:/my_images/icon/grey light.png</iconset>
   </property>
   <property name="text">
    <string>Init</string>
   </property>
  </action>
  <action name="action_Pre_OP">
   <property name="icon">
    <iconset resource="Icon_resource.qrc">
     <normaloff>:/my_images/icon/grey light.png</normaloff>:/my_images/icon/grey light.png</iconset>
   </property>
   <property name="text">
    <string>Pre-OP</string>
   </property>
  </action>
  <action name="action_Safe_OP">
   <property name="icon">
    <iconset resource="Icon_resource.qrc">
     <normaloff>:/my_images/icon/grey light.png</normaloff>:/my_images/icon/grey light.png</iconset>
   </property>
   <property name="text">
    <string>Safe-OP</string>
   </property>
  </action>
  <action name="action_OP">
   <property name="icon">
    <iconset resource="Icon_resource.qrc">
     <normaloff>:/my_images/icon/grey light.png</normaloff>:/my_images/icon/grey light.png</iconset>
   </property>
   <property name="text">
    <string>OP</string>
   </property>
  </action>
  <action name="action_Boot_Strap">
   <property name="icon">
    <iconset resource="Icon_resource.qrc">
     <normaloff>:/my_images/icon/grey light.png</normaloff>:/my_images/icon/grey light.png</iconset>
   </property>
   <property name="text">
    <string>Boot</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StayOpenMenu</class>
   <extends>QMenu</extends>
   <header location="global">stayopenmenu.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icon_resource.qrc"/>
 </resources>
 <connections/>
</ui>
